theme/templates/resource-landing-page/coverage.html

```diff
@@ -252,55 +241,51 @@ <h4>Spatial:</h4>
 
                                 <div id="div_id_westlimit" class="control-group" style="margin-bottom: 10px;">
                                     <div class="input-group">
-                                        <span class="input-group-addon coordinates-label-addon">West Longitude*</span>
+                                        <span class="input-group-addon coordinates-label-addon">West Longitude</span>
                                         <input type="number" class="form-control" placeholder="-180 to 180"
                                                id="id_westlimit" name="westlimit"
                                                value="{% if coverage_spatial_form.initial.westlimit %}{{ coverage_spatial_form.initial.westlimit|floatformat:"4" }}{% endif %}">
                                         <span data-toggle="tooltip" data-placement="auto" title="WGS 84 decimal degrees"
                                               class="input-group-addon">°</span>
                                     </div>
                                 </div>
+                                <br>
+                                {# we will show/hide this button from JS code when ever coverage element gets updated for aggregation #}
+                                <button id="btn-update-resource-spatial-coverage"
+                                        type="button" class="btn btn-info"
+                                        {% if not cm.resource_type == "CompositeResource" or not cm.has_logical_spatial_coverage %}
+                                        style="display: none"
+                                        {% endif %}>
+                                    <i class="fa fa-calculator" aria-hidden="true"></i> Set spatial coverage from content files
```

Comment:
If we have time, we can move the white space character into CSS.

Reply:
Agree. Will change too.